config/prod.exs

```diff
@@ -32,7 +32,9 @@ config :mongoose_push, MongoosePushWeb.Endpoint,
   check_origin: true,
   server: true
 
-config :mongoose_push, :logging, level: {:system, :atom, "PUSH_LOGLEVEL", :info}
+config :mongoose_push, :logging,
+  level: {:system, :atom, "PUSH_LOGLEVEL", :info},
+  format: {:system, :atom, "PUSH_LOGFORMAT", :logjson}
```

Comment:
Actually `json` is still `json` :) Sorry for confusion, but I meant that `logfmt` is a name of format, in the same as `JSON` is a name of the format. There is no such thing as `logjson` or `fmt`. The `log` part of `logfmt` is just integral part of the name, not anything generic ;)

Reply:
Oh, okay, thanks! :)